examples/minimal/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "checkout-ui-minimal",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "private": "true",
+  "scripts": {
+    "start": "NODE_ENV=development webpack-dev-server --config ./webpack.config.js",
+    "build": "NODE_ENV=production webpack --config ./webpack.config.js",
+    "serve": "cd dist && http-serve"
+  },
+  "dependencies": {
+    "@shopify/checkout-ui-react": "latest",
```

Comment:
This uses the latest version of the package, to me this is acceptable and doesn't force us to keep examples up to date when we are in between two versions. We could also simply pin the version.

Reply:
I don't even know the behavior of latest with a lockfile, but I would pin the version anyways, too.